Realme/oppo6833-SystemUI/AndroidManifest.xml

```diff
@@ -0,0 +1,9 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.realme.oppo6833.systemui"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="com.android.systemui"
+                android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
+                android:requiredSystemPropertyValue="+oppo6833*"
```

Comment:
pretty sure that's wrong

Reply:
see 9f6692c49229f0e8effcc6a8ebcc099cce88922b